src/js/profile/mobile/widget/Calendar.js

```diff
@@ -301,6 +397,12 @@
 				return element;
 			};
 
+			prototype._configure = function (element) {
+				if (element.getAttribute("data-past-selection")) {
```

Comment:
Remove this. You don't need this. Base widget is does it during creation.
This value is accessible by widget instance `this.options.pastSelection`

Reply:
oh. got it. thanks.